pkg/controller/replication/replication_controller.go

```diff
@@ -353,22 +353,24 @@ func (rm *ReplicationManager) updatePod(old, cur interface{}) {
 		return
 	}
 	curPod := cur.(*api.Pod)
-	rc := rm.getPodController(curPod)
-	if rc == nil {
-		return
-	}
 	oldPod := old.(*api.Pod)
-
+	glog.V(6).Infof("Pod %s updated, objectMeta %+v -> %+v.", curPod.Name, oldPod.ObjectMeta, curPod.ObjectMeta)
```

Comment:
V(6) is useless IMO, we need V(4) for it to show up in test logs. I think V(4) objectmeta is fine, the really verbose part are container statuses.